lib/conjur/conjur_config.rb

```diff
@@ -15,8 +15,32 @@ class ConjurConfig < Anyway::Config
       trusted_proxies: (ENV['TRUSTED_PROXIES'] || "")
     )
 
+    # Perform validations and collect errors then report results as exception
+    on_load do
+      invalid = []
+
+      invalid << "trusted_proxies" unless validate_trusted_proxies
```

Comment:
what's going on here? should this be `invalid << trusted_proxies`? seems that you are adding the string "trusted_proxies" instead of the actual `trusted_proxies` list

Reply:
We need to collect the names of the fields for the error raised at the end of the method that reports which fields failed validation.

Reply:
so what's the `invalid.join(', ')` below do? wouldn't it always be just the string "trusted_proxies" (unless it's empty)?

Reply:
For now yeah but when we start adding new configuration values we'll collect them in a list and join them into a string for the error message. Just establishing the pattern for later.

Reply:
got it, thanks for clarifying